bundle/manifests/dscinitialization.opendatahub.io_dscinitializations.yaml

```diff
@@ -189,6 +189,44 @@ spec:
                     x-kubernetes-validations:
                     - message: MonitoringNamespace is immutable
                       rule: self == oldSelf
+                  traces:
```

Comment:
we should add trace in example https://github.com/opendatahub-io/opendatahub-operator/blob/6f0313a6c4d78f428b454bb167eeaeca673f072f/README.md#example-dscinitialization ?